packages/cmf/src/store.js

```diff
@@ -105,6 +118,9 @@ function getMiddlewares(middleware) {
 	if (!defaultRouterOverwrite) {
 		setRouterMiddleware(routerMiddleware(hashHistory));
 	}
+	if (!defaultHttpMiddlewareOverwrite) {
+		setHttpMiddleware(httpMiddleware());
```

Comment:
setHttpMiddleware missing in the export default

Reply:
export is available